src/convertQwertyToAlphabet/convertQwertyToAlphabet.ts

```diff
@@ -28,4 +28,4 @@ export function convertQwertyToHangul(word: string): string {
     return '';
   }
   return assemble([...convertQwertyToAlphabet(word)]);
-}
+}
```

Comment:
마지막에 new line을 추가해주실 수 있나요?

[POSIX 표준을 지키기 위함입니다](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

```suggestion
}

```

Reply:
넵!